### ISSUE TITLE
Auth.login() does not return rejection

### ISSUE THREAD

**Issue by Coroecram** [cloudspace/angular_devise#88]
When there is are invalid Auth.login credentials the failure function is never called. Only a 401 unauthorized access is returned.

My code:
    Auth.login($scope.user).then(function(){
      $state.go('home');
    }, function(error){
      // never called
    });

**Comment by leomao10** [comment]
@Coroecram That is because you enable `AuthIntercept` for your application. And `AuthInterceptor` return a promise that haven't been rejected or resolved

In your example, you can ignore the intercepor by passing `interceptAuth: false` to login method, like this:
```javascript
Auth.login($scope.user, { interceptAuth: false }).then(function(){
  $state.go('home');
}, function(error){
  // Error handing
});
```

**Comment by TheOneTheOnlyDavidBrown** [comment]
closed in #89

**Comment by Coroecram** [comment]
@leomao10 Thank you very much, I got that from a tutorial which then acted like it still handled the errors normally.
